### ISSUE TITLE
SYNOPSIS format in man-page is not unified

### ISSUE THREAD

**Issue by miztake** [linux-audit/audit-userspace#194]
The SYNOPSIS format is different for each man-page.
The following patterns can be seen.

pattern 1:
Header file is bold.
Function prototype is not bold.
e.g.: audit_open(3)

pattern 2:
Header file is not bold.
Function prototype is only function name bold.
e.g.: set_aumessage_mode(3)

pattern 3:
Both header files and function prototypes are bold.
e.g.: audit_log_semanage_message(3)

Other patterns also exist.

The inconsistent format does not cause bugs.
Therefore, I don't consider it a serious issue.
However, readability is higher when the format is unified.

For example, glibc seems to have the following format.

Header file is bold.
Function prototype is bold, but its argument is not bold, it is underlined.
The arguments in DESCRIPTION are also underlined

**Comment by stevegrubb** [comment]
This is true. They have been modified many times by other people adding italics, bold, and various other formatting.

**Comment by stevegrubb** [comment]
Git commits b13faa3, b9c44829, and f8dea086 should mostly get us aligned on the glibc style. I'll take one more look another day and close this out if nothing is found.

**Comment by stevegrubb** [comment]
Thanks for the report. I think this can be closed out now. Feel free to reopen if this warrants more discussion.
